v5migrationhelper/orm/src/main/java/org/hibernate/search/query/hibernate/impl/FullTextQueryImpl.java

```diff
@@ -197,6 +197,10 @@ public int doGetResultSize() {
 		return hSearchQuery.getResultSize();
 	}
 
+	public long getResultCount() {
+		return doGetResultSize();
+	}
```

Comment:
we weren't testing the migration helper in the upgrade jobs, so this one slipped out, but it's a new method in ORM 6.5+ sooo would need that 🙈

Reply:
Ok, then we'll need a Jira (I think there is one, this is just a backport?) and a 7.1.1 :/

Reply:
https://hibernate.atlassian.net/browse/HSEARCH-5092

We did this for the `HibernateOrmSearchQueryAdapter` but since it's already released in 7.1.0 ... probably a new one is needed now 😔

Reply:
No that's fine, under `Actions` you have ` Backport` action to assign an issue to another Fix version.

Reply:
![image](https://github.com/hibernate/hibernate-search/assets/412878/5a649b2f-3f7b-477a-bd13-d20ae63039ab)

Reply:
Ah... 7.1.0. Right. We need another Jira.

Reply:
🙈